google/cloud/security/scanner/scanner.py

```diff
@@ -114,6 +116,8 @@ def main(_):
     # instead of its own execution path.
     if rules_engine_name == 'GroupsEngine':
         all_violations = scanner.run(FLAGS.rules)
+        for v in all_violations:
+            print _get_violation_project(v, snapshot_timestamp)
```

Comment:
I will remove it for now, since I can't find anything called _get_violation_project. We can add it later if needed